proto/v1beta1/spdx.proto

```diff
@@ -256,12 +258,138 @@ message PackageInfoOccurrence {
   // A place for the SPDX file creator to record any general
   // comments about the package being described
   string comment = 6;
+  // The type of package: OS, MAVEN, GO, GO_STDLIB, etc.
+  string package_type = 7;
+  // Identify the full name of the package as given by the Package Originator
+  string title = 8;
```

Comment:
to confirm: we'd still need this same data in the Note, correct? if you have an example of what the data should look like in the scenarios you're looking into, that'd be helpful

Reply:
The Note will still need the same data. For example:
`
// PackageInfoNote example
{
    name: "projects/test-project/notes/123",
    short_description: "jackson-databind",
    long_description: "",
    kind: "SPDX_PACKAGE",
    related_url: [{
        url: "https://search.maven.org/artifact/com.fasterxml.jackson.core/jackson-databind",
        label: "More Info"
    }],
    spdx_package: {
        title: "com.fasterxml.jackson.core:jackson-databind",
        version: "2.13.0-rc2",
        supplier: "FasterXML",
        home_page: "http://github.com/FasterXML/jackson",
        license_declared: "The Apache Software License, Version 2.0",
        summary_description: "General data-binding functionality for Jackson: works on core streaming API",
        package_type: "MAVEN"
    }
}

// PackageInfoOccurrence example
{
    name: "projects/test-project/occurrences/345",
    resource: "https://us-west1.docker.pkg.dev/test-project/my-repo/debian/sha256:12345",
    note_name: "projects/test-project/notes/123",
    kind: "SPDX_PACKAGE",
    spdx_package: {
        title: "com.fasterxml.jackson.core:jackson-databind",
        version: "2.13.0-rc2",
        checksum: "123456",
        home_page: "http://github.com/FasterXML/jackson",
        license_concluded: "The Apache Software License, Version 2.0",
        summary_description: "General data-binding functionality for Jackson: works on core streaming API",
        package_type: "MAVEN",
    }
}
`